sdk/metric/controller/basic/controller.go

```diff
@@ -57,13 +57,8 @@ var ErrControllerStarted = fmt.Errorf("controller already started")
 // be blocked by a pull request in the basic controller.
 type Controller struct {
 	// lock protects libraries and synchronizes Start() and Stop().
```

Comment:
Please update this comment to reflect the change.

```suggestion
	// lock synchronizes Start() and Stop().
```